src/js/env/node/networkUtils.js

```diff
@@ -8,13 +8,34 @@ if (global && typeof global.fetch !== 'function') {
     global.fetch = nodeFetch;
 }
 
+const connectCommonFilesPath = path.resolve(
+    __dirname,
+    '../../../../node_modules/@trezor/connect-common/files/',
+);
+
+const getAbsolutePath = (fileUrl: string) => {
+    switch (fileUrl) {
+        case './data/bridge/releases.json':
+            return path.join(connectCommonFilesPath, 'bridge/releases.json');
+        case './data/bridge/latest.txt':
+            return path.join(connectCommonFilesPath, 'bridge/latest.txt');
```

Comment:
im wondering if this file is even used (there is no such case in react-native)

Reply:
👍 deleted